server/config.go

```diff
@@ -37,6 +36,20 @@ func newAuthConfigResponse(config chronograf.Config) *authConfigResponse {
 	}
 }
 
+type logViewerUIResponse struct {
+	Links selfLinks `json:"links"`
+	chronograf.LogViewerUIConfig
+}
+
+func newLogViewerUIConfigResponse(config chronograf.Config) *logViewerUIResponse {
```

Comment:
Why is this a function of `chronograf.Config` and not `chronograf.LogViewerUIConfig`?